docker/android-sdk/tools/watch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+inotifywait -mr $HOME/.android -e create -e moved_to -e modify -e attrib |
```

Comment:
The thing that concerns me here is that the `android-sdk` Pod needs to run permanently for this solution. If this logic or something similar is required I'd prefer it to be in the slave